layouts/d1/modules/Users/DetailViewBlockView.tpl

```diff
@@ -75,7 +75,7 @@
                                                     {vtranslate($FIELD_MODEL->get('fieldvalue'),$FIELD_MODEL->get('fieldvalue'))}
                                                 {/if}
 											</span>
-                                                {if $IS_AJAX_ENABLED && $FIELD_MODEL->isEditable() eq 'true' && $FIELD_MODEL->isAjaxEditable() eq 'true'}
+                                                {if $IS_AJAX_ENABLED && $FIELD_MODEL->isEditable() eq 'true' && $FIELD_MODEL->isAjaxEditable() eq 'true' && $FIELD_MODEL->get('name') neq 'signature'}
```

Comment:
this should be added to function isAjaxEditable with module condition Users

if module = Users and field name = signature